### DISCUSSION TITLE
Feature request: add export of a list of selected file names

### DISCUSSION THREAD

**Discussion by blonchkman** [aurelienpierreeng/ansel#796] [Ideas]
**What is the current problem you are facing ?**
I always import via `add to library`, excluding JPGs that are still present in the directory. I regularly destroy rejected images in the light table. The directory cleanup must include the relevant JPGs.

**Where in your workflow does your problem occur ?**
Once the RAWs have been eliminated, a little dance in the terminal would then allow me to destroy the concerned  JPGs  : 
```
diff <( ls *{ARW,CR2,CR3,NEF} 2>/dev/null | cut -f1 -d. ) \
     <( ls *JPG | cut -f1 -d. ) \
     | sed -nE "/^> (.*)/{s//\1.JPG/;p}" \
     | xargs rm
 ```
Having a list of rejected RAWs would allow me to easily remove the relevant JPGs from the directory.

Other examples could be imagined, but there's nothing very urgent or essential here.

Kindest regards

**Comment by sahlirc** [comment]
Another case figure I face, although rare, is that in the event of incomplete exif (like lens/aperture/focal length for old lenses lenses not recognized by the camera), one can easily sort these files out in the library with queries. It would be nice to be able to export the corresponding list of images (full path) so that one can correct entries with exiftool for instance in a terminal.
